src/gui/midi.cpp

```diff
@@ -106,20 +106,31 @@ struct DB_Midi {
 	struct {
 		uint8_t buf[SYSEX_SIZE];
 		size_t used;
-		Bitu delay;
-		uint32_t start;
+		uint32_t delay; // ms
+		uint32_t start; // ms
```

Comment:
And with a max value of 4-billion and some milliseconds, these `start` and `delay` values can represent up to 46 days of possible time duration (although maybe 32-bit are needed on the DOS-midi-side?).

Reply:
Nah, 32-bits are needed only because these values are taken from SDL_GetTicks() and fed into SDL_Delay. I believe value of `delay` will never be higher than (8192 * 1.25 / 3.125 + 2) = 3.2788s, but `start` is being used continuously used, so now we know that when user selects option `midiconfig=2 delaysysex` to work around bugs in physical MT-32 rev 0 device, then we get at most of 46-days of bug-free operation.

This usecase is so niche, that I don't really care about this limitation ;)